### PR TITLE
[script.module.libmediathek3] v1.3.0

### PR DESCRIPTION
**Pull request by prof-membrane**
### Description
This is a new version of my mini framework, used by my add-ons.

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [sciprt.foo.bar] v1.0.0

### CODE REVIEW

File: script.module.libmediathek3/addon.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+	     name="libmediathek3" 
+	     version="1.3.0" 
+	     provider-name="membrane">
+  <requires>
+    <import addon="xbmc.python" version="2.25.0" />
+  </requires>
+  <extension point="xbmc.python.module" library="lib" />
+  <extension point="xbmc.addon.metadata">
+    <platform>all</platform>
+    <summary lang="en">Supplementary lib for Mediathek add-ons.</summary>
```
Comment (phil65) [inline script.module.libmediathek3/addon.xml]: please add license tag.

File: script.module.libmediathek3/lib/libmediathek3listing.py [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+		if 'airedISO8601' in d or 'airedISO8601A' in d:
+			d['aired'],d['airedtime'] = _airedISO8601(d)
+			
+		if 'type' in d and d['type'] == 'date' and 'airedtime' in d:
+			d["name"] = '(' + str(d["airedtime"]) + ') ' + d["name"]
+		elif 'type' in d and d['type'] == 'date' and 'time' in d:
+			d["name"] = '(' + str(d["date"]) + ') ' + d["name"]
+			
+		ilabels = {
+			"Title": clearString(d.get('name','')),
```
Comment (phil65) [inline script.module.libmediathek3/lib/libmediathek3listing.py]: please also set mediatype.

Comment (prof-membrane) [inline script.module.libmediathek3/lib/libmediathek3listing.py]: Sure thing - but what is it good for?

File: script.module.libmediathek3/addon.xml (hunk 1 above)
Comment (prof-membrane) [inline script.module.libmediathek3/addon.xml]: Roger.

File: script.module.libmediathek3/lib/libmediathek3listing.py (hunk 2 above)
Comment (phil65) [inline script.module.libmediathek3/lib/libmediathek3listing.py]: mainly for skins, and other add-ons might depend on it too.
just fyi, perhaps worth reading: http://forum.kodi.tv/showthread.php?tid=299107

**Comment by prof-membrane** [comment]
Both issues should be resolved now.

File: script.module.libmediathek3/lib/libmediathek3listing.py [hunk 3]
```diff
@@ -89,24 +100,22 @@ def addEntries(l):
...
-				lists.append([u,liz,True])
-			else:
-				xbmcplugin.setContent( handle=int( sys.argv[ 1 ] ), content="files" )
-				lists.append([u,liz,True])
+		
+		#if d['type'] == 'clip':
+		if False:
+			xbmc.log('ignoring clip')
+		elif d.get('type',None) == 'video' or d.get('type',None) == 'live' or d.get('type',None) == 'date' or d.get('type',None) == 'clip' or d.get('type',None) == 'episode':
+			xbmcplugin.setContent( handle=int( sys.argv[ 1 ] ), content="episodes" )
```
Comment (phil65) [inline script.module.libmediathek3/lib/libmediathek3listing.py]: you are calling setContent() for every single listitem? You only need to call it once for each folder listing.

**Comment by prof-membrane** [comment]
Done.

**Comment by phil65** [comment]
Thx for your patience, all lookin good to me now.

**Pull request merged**